vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -93,7 +90,6 @@ $global:map = @{
     # specified by AKS PR for most of the cases. BUT as long as there's a new unpacked image version, we should keep the
     # versions synced.
     "c:\akse-cache\containerd\"   = @(
-        "https://acs-mirror.azureedge.net/containerd/windows/v0.0.41/binaries/containerd-v0.0.41-windows-amd64.tar.gz",
```

Comment:
Just curious, why removed the old cached files, for a regular clean up or do we have no enough space in VHD?

Reply:
I tried to clean up unused cached files when adding new packages. If we always keep to remove unused cached files, it should avoid us to hit the issue no enough space.